test/unit/math/prim/scal/fun/digamma_test.cpp

```diff
@@ -15,4 +15,8 @@ TEST(MathFunctions, digamma_nan) {
   EXPECT_PRED1(boost::math::isnan<double>, stan::math::digamma(nan));
 
   EXPECT_PRED1(boost::math::isnan<double>, stan::math::digamma(-1));
+
+  // see comments in test/unit/math/prim/scal/fun/lgamma_test.cpp
+  EXPECT_PRED1(boost::math::isnormal<double>,
+               boost::math::digamma(1.0E50, stan::math::boost_policy_t()));
```

Comment:
Should this be testing stan’s digamma instead of boost’s?

Reply:
both should work... if you prefer to test stan::math::digamma then we can do that.